src/test/java/teammates/test/cases/browsertests/InstructorHomePageUiTest.java

```diff
@@ -222,6 +223,36 @@ private void testCourseLinks() {
 
     }
 
+    private void testDownloadAction() {
+
+        // Test that download result button exist in homePage
+        homePage.verifyDownloadResultButtonExists(feedbackSessionClosed.getCourseId(),
+                feedbackSessionClosed.getSessionName());
+
+        ______TS("Typical case: download report");
+
+        AppUrl reportUrl = createUrl(Const.ActionURIs.INSTRUCTOR_FEEDBACK_RESULTS_DOWNLOAD)
+                .withUserId("CHomeUiT.instructor.tmms")
+                .withCourseId(feedbackSessionClosed.getCourseId())
+                .withSessionName(feedbackSessionClosed.getSessionName());
+
+        homePage.verifyDownloadLink(reportUrl);
+
+        ______TS("Typical case: download report unsuccessfully");
```

Comment:
Write the reasoning here as well as to why the download report will be unsuccessful